### PR TITLE
[IMP] website: introduce `s_company_team_detail` snippet

### PR DESCRIPTION
**Pull request by mano-odoo**
This PR adds the new `s_company_team_detail` snippet.

task-4156290
Part-of: task-4077427

Design-theme: https://github.com/odoo/design-themes/pull/896

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Nice @mano-odoo some small comments

File: addons/website/views/snippets/s_company_team_detail.xml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-4 pt32 pb32" data-name="Team Member">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img class="o_editable_media img img-fluid rounded-circle" src="/web/image/website.s_company_team_image_1" alt="" style="width: 25% !important;"/>
+                    </div>
+                    <p>
+                        <br/>
+                    </p>
+                    <h4 class="h5-fs">James Mitchell</h4>
+                    <p class="text-muted">Chief Technical Officer (CTO)</p>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_company_team_detail.xml:18]: Also, nitpicking but I think it's weird to have only one role with something inside parenthesis `(CTO)`

File: addons/website/views/snippets/s_company_team_detail.xml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                    <p>James leads the tech strategy and innovation efforts at the company.</p>
+                    <div class="s_social_media o_not_editable text-start" data-snippet="s_social_media" data-name="Social Media">
+                        <h4 class="s_social_media_title d-none o_default_snippet_text">Social Media</h4>
```
[the remark this answers is not in the document]
Comment (mano-odoo) [reply addons/website/views/snippets/s_company_team_detail.xml:21]: I will replace with `<h5>` and leave it unresolved for tech review as I'm unsure it's risk free to remove the title (and the option). 

Maybe we should even keep the `<h4>` which come with the s_social snippet and apply a `h5-fs` class to be even more safe 🤷

File: addons/website/views/snippets/s_company_team_detail.xml [hunk 3]
```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_detail" name="Team Detail">
+    <section class="s_company_team_detail o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <h3>Meet our team</h3>
+            <p class="lead">Dedicated professionals driving our success</p>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_company_team_detail.xml:8]: This is usually what we do with heading + lead. It's better structured, easier to resize, you can play with paddings...
```suggestion
        <div class="col-lg-12">
            <h3>Meet our team</h3>
            <p class="lead">Dedicated professionals driving our success</p>
        </div>
```

**Review by anso-odoo — APPROVED** [review]
Neat, good job!

Just one small comment, otherwise LGTM

Let's keep an eye on the task to add keywords to the PR when we have them.

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
Nice snippet :+1: 
Here is my review

File: addons/website/views/snippets/s_company_team_detail.xml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            </div>
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-4 pt32 pb32" data-name="Team Member">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img class="o_editable_media img img-fluid rounded-circle" src="/web/image/website.s_company_team_image_1" alt="" style="width: 25% !important;"/>
+                    </div>
+                    <h4 class="h5-fs"><br/>James Mitchell</h4>
+                    <p class="text-muted">Chief Technical Officer</p>
+                    <p>James leads the tech strategy and innovation efforts at the company.</p>
+                    <div class="s_social_media o_not_editable text-start" data-snippet="s_social_media" data-name="Social Media">
```
[the remark this answers is not in the document]
Comment (mano-odoo) [reply addons/website/views/snippets/s_company_team_detail.xml]: Indeed, let's use the `no_icon_color` by default

**Review by bvr-odoo — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**